metadata.json

```diff
@@ -26,7 +26,7 @@
     },
     {
       "name": "puppet/systemd",
-      "version_requirement": ">= 7.0.0 <= 8.0.0"
+      "version_requirement": ">= 7.0.0 <= 9.0.0"
```

Comment:
IMHO this should be `< 9.0.0`, otherwise you allow the breaking 9.0.0 already, but no other 9.x release.
(this applies to all dependencies)

Reply:
you're right